geometry.dtx

```diff
@@ -3395,13 +3395,25 @@ the derived files: geometry.{sty,ins,drv}, geometry-samples.tex.
 %
 %    Macros for the page frames and cropmarks.
 %    \begin{macrocode}
-\def\Gm@cropmark(#1,#2,#3,#4){%
-  \begin{picture}[artifact](0,0)
-    \setlength\unitlength{1truemm}%
-    \linethickness{0.25pt}%
-    \put(#3,0){\line(#1,0){17}}%
-    \put(0,#4){\line(0,#2){17}}%
-  \end{picture}}%
+\IfDocumentMetadataTF
```

Comment:
This is needed but it probably shouldn't be, shouldn't we make picture always have a no-op argument as we have done for the equivalent keys on `\includegraphics` ? (not a question for this PR)